### PR TITLE
feat(getStatus): enable consumers to programmatically query statuses

### PR DESCRIPTION
**Pull request by gergelyke**

### CODE REVIEW

File: lib/terminus.js [hunk 1]
```diff
@@ -125,6 +129,9 @@ function terminus (server, options = {}) {
     })
   }
 
+  // add the healthcheck to the internal states
+  internal.healthChecks = healthChecks
```
Comment (BrunnerLivio) [inline lib/terminus.js]: If calling createTerminus multiple times, wouldn't this just override the last health checks?

Comment (gergelyke) [reply lib/terminus.js]: oh, you are right 🤔

Comment (gergelyke) [reply lib/terminus.js]: reopening, sorry for closing this convo

Comment (BrunnerLivio) [reply lib/terminus.js]: No prob haha :)
I really do not think we get around a class as I proposed in my PR..

**Pull request closed without merging**